deepparse/parser/address_parser.py

```diff
@@ -156,6 +166,8 @@ def __call__(self,
         # since training data is lowercase
         lower_cased_addresses_to_parse = [address.lower() for address in addresses_to_parse]
 
+        if self.verbose and len(addresses_to_parse) > 64:
```

Comment:
```python
and len(addresses_to_parse) > 64
```
Why?

Reply:
Yeah, it's not clear. It's the threshold where the time starts to be more of a concern.

Reply:
I've added a global variable with more information about it.
#dc3b619